test/ninja/generate_and_build.py

```diff
@@ -49,10 +49,11 @@
 test.file_fixture('ninja_test_sconscripts/sconstruct_generate_and_build', 'SConstruct')
 
 # generate simple build
-test.run(stdout=None)
+test.run(stdout=None, arguments='NINJA_CMD_ARGS=-v')
```

Comment:
I just hooked into another test since this was a smaller change and I didn't want to create an entirely new test to run for it.

Reply:
k.